app/src/main/java/io/homeassistant/companion/android/settings/SettingsFragment.kt

```diff
@@ -125,6 +127,17 @@ class SettingsFragment : PreferenceFragmentCompat(), SettingsView {
             return@setOnPreferenceClickListener true
         }
 
+        findPreference<Preference>("changelog")?.let {
+            val link = if (BuildConfig.VERSION_NAME.startsWith("LOCAL"))
+                "https://github.com/home-assistant/android/releases"
+            else "https://github.com/home-assistant/android/releases/tag/${BuildConfig.VERSION_NAME}"
+            it.summary = link
+            val intent = Intent()
+            intent.action = "android.intent.action.VIEW"
+            intent.data = Uri.parse(link)
+            preferenceScreen.findPreference<Preference>("changelog")!!.intent = intent
```

Comment:
Can't this just be this?
```suggestion
            it.intent = intent
```

Reply:
yup, not sure why I thought I had to do that :P